### PR TITLE
SSAO component documentation

### PR DESCRIPTION
**Pull request by antonmic**
Signed-off-by: antonmic <56370189+antonmic@users.noreply.github.com>

### CODE REVIEW

**Review by galibzon — APPROVED** [review]

File: content/docs/user-guide/components/reference/atom/ssao.md [hunk 1]
```diff
@@ -5,16 +5,25 @@ description: 'Open 3D Engine (O3DE) SSAO component reference.'
...
+| **Enable SSAO** | Whether SSAO should be enabled or not (note that SSAO is enabled by default - even if there is no SSAO component in the level - and an SSAO component must be added and this value set to false in order to disable SSAO). |
+| **SSAO Strength** | The strength of the SSAO effect - the higher this value the darker the SSAO will appear. |
+| **Sampling Radius** | The sampling radius of the SSAO effect in screen UV space - decrease/increase this value to have a more local/global SSAO effect. |
+| **Enable Blur** | Whether a blur is applied to the computed SSAO buffer. Enabling with result in a smoother looking SSAO image. |
+| **Blur Strength** | How strong of a blur to apply to the SSAO. Weaker values will result in a sharper albeit more noisy image. |
+| **Blur Sharpness** | The SSAO blur tries to respect geometric edges to avoid smearing the SSAO across edges or objects. Increasing this value will result in a sharper looking blur, whereas decreasing it will result in a softer looking SSAO blur. |
+| **Blur Edge Threshold** | An edge threshold can be specified for the blur to ignore depth changes of a certain value. This has the most effect at small values. |
```
Comment (chanmosq) [inline content/docs/user-guide/components/reference/atom/ssao.md:24]: "can be specified" - Does that mean we can not choose to not activate this, even with blur enabled? 

Current suggestion to rephrase assumes that it's activated if blur is enabled. And I assume that at `0.0`, it has the most effect?
```suggestion
| **Blur Edge Threshold** | An edge threshold, in which the blur ignores depth changes at the specified value. This has the most effect with smaller values. |
```

File: content/docs/user-guide/components/reference/atom/ssao.md [hunk 2]
```diff
@@ -5,16 +5,25 @@ description: 'Open 3D Engine (O3DE) SSAO component reference.'
...
+| **Enable Downsample** | Whether to downsample the depth buffer and perform SSAO and blur on the downsampled result before upsampling. This is an optimization that results in less detailed SSAO. |
```
Comment (chanmosq) [inline content/docs/user-guide/components/reference/atom/ssao.md:25]:
```suggestion
| **Enable Downsample** | If enabled, first downsamples the depth buffer and performs SSAO and blur on the downsampled result, then upsamples the result. This is an optimization that results in a less detailed SSAO. |
```

**Review by chanmosq — CHANGES_REQUESTED** [review]

**Review by micronAMZN — CHANGES_REQUESTED** [review]
Agree with @chanmosq suggestions. Also, the property table should have Value and Default columns. See other component docs for examples.

**Review by chanmosq — APPROVED** [review]

**Review by micronAMZN — APPROVED** [review]

File: content/docs/user-guide/components/reference/atom/ssao.md [hunk 3]
```diff
@@ -5,16 +5,27 @@ description: 'Open 3D Engine (O3DE) SSAO component reference.'
...
 
 
 ## Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
+{{< image-width "/images/user-guide/components/reference/atom/ssao-component.jpg" "500" "SSAO Component Properties" >}}
+
+| Property | Description | Values | Default
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/ssao.md:22]:
```suggestion
| Property | Description | Values | Default |
```

**Review by micronAMZN — APPROVED** [review]

**Pull request merged**